scss/atoms/buttons/_buttons.scss

```diff
@@ -0,0 +1,102 @@
+/*
+* ==========================================================================
+* Buttons
+* ==========================================================================
+*/
+@import "mixins";
+
+/* These colors won't remain here, will be removed with jsass  */
+$background: #fff;
+$chassis-blue: #4fc0c8;
+$chassis-gray-light: #f2f2f2;
+$chassis-gray-dark: #383838;
+
```

Comment:
Yeah, I should switch these to stuff like $button-primary-background, and $button-primary-color.  That was something to put in their temporarily, but you're right, I should have better variable names.